487 docs   expliquer le moteur de jeu (#603)
JeremieLeymarie/story-builder: documentation/docs/Builder/builder_random_events.mdx

---
sidebar_position: 5
---

# Random Events

> Random Events is a particular functionality available to you in the builder section.

Reviewer: ```suggestion
Random Events is a particular feature available to you in the builder section.
```

> Similarly to conditional actions, random events are created to dyanmize the story further and make it more alive.

Reviewer: ```suggestion
Similarly to conditional actions, random events are created to dynamize the story further and make it more alive.
```

> The way is works is actually very simple, you just need to draw two new scenes from the same scene action endpoint as pictured in the image below :

Reviewer: ```suggestion
The way it works is actually very simple, you just need to draw two new scenes from the same scene action endpoint as pictured in the image below :
```

<img src="/img/builder_random_events.png" alt="builder conditionals" width="700" />

> Here we have drawn to paths from the same action endpoint which results in the creation of a conditional action.

Reviewer: ```suggestion
Here we have drawn two paths from the same action endpoint which results in the creation of a randomized event.
```

> The way it works is quite simple all you have to do is fill the percentage of "chance" that the player will have of accessing a given scene.

Reviewer: ```suggestion
The way it works is quite simple: all you have to do is fill the percentage of "chance" that the player will have of accessing a given scene.
```

> For example let's say that i create those 2 new scenes and i give them the name "eight scene" and "ninth scene" and that when i choose the seventh action of my fourth scene i want 70%

Reviewer: ```suggestion
For example let's say that I create those 2 new scenes and I give them the name "eighth scene" and "ninth scene" and that when I choose the seventh action of my fourth scene i want 70%
```

> chance of reaching the eight scene and 30% chance of reaching the ninth scene.

Reviewer: ```suggestion
chance of reaching the eighth scene and 30% chance of reaching the ninth scene.
```

<img src="/img/builder_random_events_settings.png" alt="builder conditionals settings" width="700" />

> The choice will now be set by the user to the player of the story because each time the seventh action gets picked, the player has a 70% of accessing the eight scene against 30% for the ninth scene.

Reviewer: ```suggestion
The choice will now be set by the user to the player of the story because each time the seventh action gets picked, the player has a 70% of accessing the eighth scene against 30% for the ninth scene.
```
